services/prometheus/scrape_configs_test.go

```diff
@@ -792,6 +792,98 @@ func TestScrapeConfig(t *testing.T) {
 			require.EqualError(t, err, "failed to decode custom labels: unexpected end of JSON input")
 		})
 	})
+
+	t.Run("scrapeConfigsForRDSExporter", func(t *testing.T) {
+		t.Run("Normal", func(t *testing.T) {
+			node1 := &models.Node{
+				NodeID:    "/node_id/node1",
+				NodeType:  models.RemoteRDSNodeType,
+				NodeName:  "rds1",
+				Address:   "rds-mysql57",
+				NodeModel: "db.t3.micro",
+				Region:    pointer.ToString("us-east-1"),
+				AZ:        "us-east-1b",
+			}
+			agent1 := &models.Agent{
+				AgentID:      "/agent_id/agent1",
+				AgentType:    models.RDSExporterType,
+				PMMAgentID:   pointer.ToString("pmm-server"),
+				NodeID:       pointer.ToString("/node_id/node1"),
+				AWSAccessKey: pointer.ToString("AKIAIOSFODNN7EXAMPLE"),
```

Comment:
I hope it's not real keys :)

Reply:
Note the suffix `EXAMPLE` :) They are from https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys